package.json

```diff
@@ -5,7 +5,8 @@
   "description": "End to end tests for a Form Builder form",
   "scripts": {
     "lint": "eslint .",
-    "test": "npm run lint && ava --reset-cache && ava -v",
+    "test": "ava --reset-cache && ava -v",
```

Comment:
Why remove linting from test?

Reply:
👍 I'd have restructured the circle-ci stuff as I prefer the main `test` to run everything, but that's just me…